backend/app/usecase/shortlink/creator_test.go

```diff
@@ -51,7 +52,9 @@ func TestShortLinkCreatorPersist_CreateShortLink(t *testing.T) {
 			user: entity.User{
 				Email: "alpha@example.com",
 			},
-			shortLink: entity.ShortLink{},
+			shortLink: entity.ShortLink{
```

Comment:
If the long link is not provided, this test case will have an error because the long link is not provided. If any long link is provided then the test case has an error because the alias exist as intended!